test/e2e/common/pods.go

```diff
@@ -917,7 +919,7 @@ var _ = framework.KubeDescribe("Pods", func() {
 		framework.ExpectNoError(err, "failed to create Pod %v in namespace %v", testPod.ObjectMeta.Name, testNamespaceName)
 
 		ginkgo.By("watching for Pod to be ready")
-		ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+		ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
```

Comment:
please refactor this into a named constant or var

Reply:
It was the timeout value I had used in [e2e test: should delete a collection of pod templates](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/common/podtemplates.go#L38). The latest flake had multiple "throttling" log messages but couldn't see a single reason for it.